### ISSUE TITLE
Allow to show containing remote tracking branches for commits?

### ISSUE THREAD

**Issue by michael-heerdegen** [magit/magit#3487]
Hello,

when I hit RET on a commit in the log buffer, the overview buffer you get includes a line of "Containing:" branches.  In my case, I wanted this line to include containing remote tracking branches also.  Seems Magit currently cannot easily be configured to do so.  Would it make sense to allow a customization of this (and related) line(s)?

For now, I use the following extremely ugly hack:

```
(advice-add 'magit-git-lines :filter-args
            (defun my-magit-git-lines--filter-args-ad (args)
              (pcase args
                (`("branch" nil "--contains" . ,rest)
                 `("branch" nil "-a" "--contains" . ,rest))
                (_ args))))
```

Thanks, Michael.

**Comment by tarsius** [comment]
> Would it make sense to allow a customization of this (and related) line(s)?

Feel free to open a pull-request.

**Comment by michael-heerdegen** [comment]
I can't promise I do that soon.  I'm currently quite busy with my own stuff (el-search).  So if you want to go ahead...

**Comment by tarsius** [comment]
This would also increase the odds that not all branches can be shown on a single line, so we would also have to allow (optionally) listing them on multiple lines or something like that.

**Comment by michael-heerdegen** [comment]
Michael Heerdegen <michael_heerdegen@web.de> writes:

> > Closed #3487 via 50e05e4.
>
> Thanks, will try.

Looks good and seems to work nice for me so far.

Michael.
